VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift

```diff
@@ -86,15 +85,13 @@ struct VaultDetailView: View {
     var list: some View {
         List {
             ForEach(viewModel.coinsGroupedByChains.sorted(by: {
-                $0.order < $1.order
+                $0.totalBalanceInFiatString > $1.totalBalanceInFiatString
```

Comment:
I don't think sort by string is correct , because 2 will be larger than 10, is that right?

Reply:
Oh no, this is the other way around. The sorting order is in descending order. Chain with higher balance will be on the top.

Reply:
Let me fix that.